include/rapidjson/internal/dtoa.h

```diff
@@ -41,7 +41,7 @@ inline void GrisuRound(char* buffer, int len, uint64_t delta, uint64_t rest, uin
     }
 }
 
-inline unsigned CountDecimalDigit32(uint32_t n) {
+inline int CountDecimalDigit32(uint32_t n) {
```

Comment:
I am not sure why these are needed to be changed to signed. Number of digits must be >= 0.